packages/assets-controllers/src/NftController.test.ts

```diff
@@ -2245,14 +2450,24 @@ describe('NftController', () => {
     });
 
     it('should throw an error if selected address is not owner of input NFT', async () => {
-      const { nftController, triggerPreferencesStateChange } =
-        setupController();
+      const {
+        nftController,
+        getInternalAccountMock,
+        triggerPreferencesStateChange,
+        triggerSelectedAccountChange,
+      } = setupController();
+      // TODO: Replace `any` with type
```

Comment:
Which `any` are we talking about here? 😅

Reply:
this can be removed, it was left over from a merge